pkg/volume/csi/expander.go

```diff
@@ -94,12 +94,30 @@ func (c *csiPlugin) nodeExpandWithClient(
 		return false, nil
 	}
 
-	volumeTargetPath := resizeOptions.DeviceMountPath
+	pv := resizeOptions.VolumeSpec.PersistentVolume
```

Comment:
looks like we have 2 things named `resizeOptions` now. This one that has `VolumeSpec` that already exists and the new struct. Is there a way to disambiguate the naming?

Reply:
I named csi resize options to be `csiResizeOptions` to disambiguate. Should be enough?